pkg/clusteragent/admission/mutate/config.go

```diff
@@ -76,6 +85,21 @@ var (
 		Name:  dogstatsdURLEnvVarName,
 		Value: config.Datadog.GetString("admission_controller.inject_config.dogstatsd_socket"),
 	}
+
+	singleStepInstrumentationInstallTypeEnvVar = corev1.EnvVar{
+		Name:  instrumentationInstallTypeEnvVarName,
+		Value: singleStepInstrumentationInstallType,
+	}
+
+	localLibraryInstrumentationInstallTypeEnvVar = corev1.EnvVar{
+		Name:  instrumentationInstallTypeEnvVarName,
+		Value: localLibraryInstrumentationInstallType,
+	}
+
+	instrumentationInstallTimeEnvVar = corev1.EnvVar{
+		Name:  instrumentationInstallTimeEnvVarName,
+		Value: strconv.FormatInt(time.Now().Unix(), 10),
```

Comment:
setting the value with `time.Now()` here will always return the time of the cluster-agent start. 
I don't think it is was is expected. I suppose we want the timestamp when the envvar is injected.

Reply:
I don't know if it makes huge difference. We talked about DD_INSTRUMENTATION_INSTALL_TIME as the time when installer was started. In K8s context the installer is cluster-agent, so I think it's OK to use cluster agent start time. @Kyle-Verhoog what do you think?

Reply:
We want to measure the time between intent to install APM and when they receive the first trace. To approximate the intent we use the signal of installation. So we're aiming to generate the timestamp in the helm chart/operator to pass here, if that isn't available for some reason then the startup of the Cluster Agent is the next best approximation.

Reply:
ok it might be difficult to implement it in the helm chart too, because each time the chart will be updated the timestamp might change and so force a redeploy of the cluster-agent, which is not what we want. We only want to trigger a rollout if a configuration change.